CONTRIBUTING.md

```diff
@@ -212,14 +212,14 @@ Before release:
 
 Run the following release commands, replacing `7.0.0` with your version number:
 
-- create a release branch, replacing `xx` with your initials to make it clear who created the branch.
-  - `git checkout origin/master -b xx/release-7.0.0`
+- create a release branch
```

Comment:
... and switch to it. All building, tagging and publishing will be done on the release branch, and then merged back into master in a later step.